### PR TITLE
 translate registering-a-test-device.md into japanese

### PR DESCRIPTION
**Pull request by kenjitayama**
Hi! I translated `_articles/testing/registering-a-test-device.md`.
I translated while actually following the steps :)

Some parts that I think should be fixed in the original are:

1. `[Test Devices](https://www.bitrise.io/me/profile#/test_devices)` : unnecessary back tick 
1. I changed `Click` to `Tap` (タップ) where it is talking about Safari on a iOS

### CODE REVIEW

File: _articles/translations/testing/registering-a-test-device.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: テストデバイスの登録
+menu:
+  testing:
+    weight: 1
+
+---
```
Comment (zoltan-baba) [inline _articles/translations/testing/registering-a-test-device.md]: Hi! Please remove the "menu" part of the frontmatter! The title can and should stay, preferably, but the "menu" element should be removed. Thanks!

**Comment by zoltan-baba** [comment]
Thanks for the feedback, I've fixed the stuff! And of course thanks for your effort, great work!

**Comment by kenjitayama** [comment]
Glad the feedback to the original was worthwhile too :)
I removed the "menu" element. Is this how it should be?

**Comment by zoltan-baba** [comment]
Yes, thank you! Great work, we really appreciate it!

**Pull request merged**